improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -1095,6 +1107,103 @@ def test_coefficients_truncnorm_realizations_no_statsmodels(self):
             [cube.name() for cube in result], self.expected_coeff_names
         )
 
+    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_each_point(self):
+        """Test computing coefficients independently for each grid point (initial guess
+        and minimising) returns the expected coefficients and associated metadata."""
+        plugin = Plugin(self.distribution, each_point=True)
+        result = plugin.process(
+            self.historic_temperature_forecast_cube, self.temperature_truth_cube
+        )
+        for cube in result:
+            self.assertEMOSCoefficientsAlmostEqual(
+                cube.data, self.expected_mean_predictor_each_grid_point[cube.name()],
+            )
+            self.assertIn(cube.name(), self.expected_coeff_names)
+            self.assertEqual(
+                [c.name() for c in cube.coords(dim_coords=True)],
+                ["latitude", "longitude"],
+            )
+
+    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_each_point_sites(self):
+        """Test computing coefficients independently for each site location
+        (initial guess and minimising) returns the expected coefficients and
+        associated metadata."""
+        plugin = Plugin(self.distribution, each_point=True)
+        result = plugin.process(self.historic_forecast_spot_cube, self.truth_spot_cube)
+        for cube in result:
+            self.assertEMOSCoefficientsAlmostEqual(
+                cube.data, self.expected_mean_predictor_each_site[cube.name()],
+            )
+            self.assertIn(cube.name(), self.expected_coeff_names)
+            self.assertEqual(
+                [c.name() for c in cube.coords(dim_coords=True)], ["spot_index"],
+            )
+
+    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_each_point_sites_realizations(self):
+        """Test computing coefficients independently for each site location
+        (initial guess and minimising) using realizations as the predictor
+        returns the expected coefficients and associated metadata."""
+        expected_dim_coords = {
+            "emos_coefficient_alpha": ["spot_index"],
+            "emos_coefficient_beta": ["realization", "spot_index"],
+            "emos_coefficient_gamma": ["spot_index"],
+            "emos_coefficient_delta": ["spot_index"],
+        }
+
+        plugin = Plugin(self.distribution, predictor="realizations", each_point=True)
+        plugin.ESTIMATE_COEFFICIENTS_FROM_LINEAR_MODEL_FLAG = False
```

Comment:
This does not need to be set as the plugin does this when it receives realizations as the predictor.

Reply:
Removed.